src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -0,0 +1,45 @@
+package blackjack.view
+
+import blackjack.model.Dealer
+import blackjack.model.DealerResult
+import blackjack.model.Participant
+import blackjack.model.PlayerResult
+import blackjack.model.Players
+
+object OutputView {
+    fun displayPlayerNames(players: Players) {
+        println(players.players.joinToString())
+    }
+
+    fun displayInitialCardsMessage(players: Players) {
+        println("\nDealing two cards to dealer, ${players.players.joinToString()}.")
+    }
+
+    fun displayAllCardsMessage(participant: Participant) {
+        print(participant.name + "'s cards: ")
+        println(participant.cardsToString())
```

Comment:
```suggestion
    println("${participant.name}'s cards: ${participant.cardsToString()}")

```

I think we can achieve the same output with less `println` method call by refactoring like above.
Also, if we need to print multiple lines, we can use multi-line string feature that Kotlin provides.

Why don't we use them to refactor our view related logic?

Reply:
Yes, I will try to reduce the calls to `println`.